frontend/packages/console-dynamic-plugin-sdk/src/coderefs/coderef-resolver.ts

```diff
@@ -97,7 +97,7 @@ export const resolveEncodedCodeRefs = (
   extensions: Extension[],
   entryModule: RemoteEntryModule,
   pluginID: string,
-  errorCallback: VoidFunction,
+  errorCallback?: VoidFunction,
```

Comment:
Rather than pass-through of an optional `errorCallback` argument, I'd suggest using a sensible default here:

```ts
errorCallback: VoidFunction = _.noop,
```

This way, we don't need to change `errorCallback();` calls in `loadReferencedObject` function above.

Reply:
IMO, it is safer to use the optional arg and optional chaining. If we set a default arg, it's still possible to pass `null` into this function and cause it to throw an exception. This approach also reduces dependency on lodash.

Reply:
That being said, I don't think this change is necessary for the sake of this bug fix, so I might remove it regardless.

Reply:
Yeah, I was actually wondering why we need to make the error callback optional. If we don't need this change, I'd just remove it.